cli/src/action/database/upgrade/node_id.rs

```diff
@@ -201,7 +201,7 @@ mod tests {
     #[test]
     // Test reading from the node_id file works as intended.
     fn test_migrate_to_db_with_file() {
-        let directory = TempDir::new("test").expect("could not create temp directory");
+        let directory = TempDir::new().expect("could not create temp directory");
```

Comment:
Keeping the prefix here could be done with this library:

```
let directory = Builder::new().prefix("test").tempdir().expect(...);
```

Reply:
Updated!